plugins/Hosting/js/sagas/saga.js

```diff
@@ -52,14 +52,25 @@ function *announceHost(action) {
 	}
 }
 
+// bytesToStorage converts a BigNumber of GB to a valid size of storage, in
+// bytes, rounded to the nearest 256MiB (64 * SectorSize).
+const bytesToStorageBytes = (bytes) => {
+	const roundedBytes = bytes.minus(bytes.modulo(2.6843546e8))
+	if (roundedBytes.isNegative()) {
+		return '0'
+	}
```

Comment:
it turns out that it is not possible for this to be negative, since the UI enforces a minimum size of 35GB in the dialogs. I wanted to program this as defensively as possible, though.